js.go

```diff
@@ -187,6 +187,9 @@ const (
 
 	// Default number of retries
 	DefaultPubRetryAttempts = 2
+
+	// defaultAsyncPubAckInflight is the number of async pub acks inflight.
+	defaultAsyncPubAckInflight = 4 * 1024
```

Comment:
FYI: For server change and max ack pending, I was going for 1024, but then realize that we maybe need to stop the kilo byte semantic for just numbers. So I set it to 1000 instead. But will see if Derek asks me to set to 1024 there, so don't update just yet :-)

Reply:
I do everything old school and power of 2, but let's do what we think is right going forward.